obspy/io/ah/core.py

```diff
@@ -228,6 +227,219 @@ def _unpack_trace(data):
         return st
 
 
+def _write_ah1(stream, filename):
+    """
+    Writes a Stream object to an AH v1 waveform file.
+
+    :type stream:
+    :param stream: The ObsPy Stream object to write.
+    :type filename: str
+    :param filename: open file, or file-like object
+
+    """
+    packer = xdrlib.Packer()
+
+    for tr in stream:
+        if hasattr(tr.stats, 'ah'):
+            packer = _pack_trace_with_ah_dict(tr, packer)
+        else:
+            packer = _pack_trace_wout_ah_dict(tr, packer)
+
+    with open(filename, 'wb') as fh:
+        fh.write(packer.get_buffer())
+
+
+def _pack_trace_with_ah_dict(tr, packer):
+
+    # station info
+    packer.pack_int(6)
+    packer.pack_fstring(6, tr.stats.ah.station.code.encode('utf-8'))
+    packer.pack_int(6)
+    try:
+        packer.pack_fstring(6, tr.stats.channel.encode('utf-8'))
+    except:
+        packer.pack_fstring(6, tr.stats.ah.station.channel.encode('utf-8'))
+
+    packer.pack_int(8)
+    packer.pack_fstring(8, tr.stats.ah.station.type.encode('utf-8'))
+    packer.pack_float(tr.stats.ah.station.latitude)
+    packer.pack_float(tr.stats.ah.station.longitude)
+    packer.pack_float(tr.stats.ah.station.elevation)
+    packer.pack_float(tr.stats.ah.station.gain)
+    packer.pack_float(tr.stats.ah.station.normalization)
+
+    poles = tr.stats.ah.station.poles
+    zeros = tr.stats.ah.station.zeros
+
+    # Poles and Zeros
+    packer.pack_float(len(poles))
+    packer.pack_float(0)
+    packer.pack_float(len(zeros))
+    packer.pack_float(0)
+
+    for _i in range(1, 30):
+        try:
+            r, i = poles[_i].real, poles[_i].imag
+        except IndexError:
+            r, i = 0, 0
```

Comment:
@s-schneider this looks really odd.. I assume this `0, 0` is supposed to be a place holder for "no pole"/"no zero"?